index.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <div class="div-container">
+      <section>
+        <header class="header">
+          <div class="My-Logo">
+            <h1>My Logo</h1>
+          </div>
+          <div class="header-Line-parent">
+            <div class="header-Line-1"></div>
+            <div class="header-Line-2"></div>
+            <div class="header-Line-3"></div>
+          </div>
+        </header>
+      </section>
+
+      <section class="section-1">
+        <div class="heading-1">
+          <h1 class="First-heading">
+            I'm Mariane <br />
+            Glad to see <br />
+            you!
+          </h1>
+          <span class="First-heading-description">
+            I'm a software developer! I can help you build a prduct,feature or
+            website Look through some of my work and experience! if you like
+            what you see and have a project you need coded, don't hesitate to
+            contact me.
+          </span>
```

Comment:
- I'd like to kindly suggest that you wrap this text in a `<p>` tag. This way will be semantically correct and the screen readers and other technologies will thank you. 👍🏽